internal/defaultchecks/default_checks.go

```diff
@@ -8,5 +8,9 @@ var (
 	// List is the list of built-in checks that are enabled by default.
 	List = set.NewFrozenStringSet(
 		"privileged-container",
+		"env-var-secret",
+		"no-v1beta-deployments",
```

Comment:
Does this one already exist?

Reply:
Ah, good catch. That's on another branch now, initially had it here, but decided to refactor it before creating a PR. Removed it from this list and added a unit test to make sure all these check names are valid.